init.sls

```diff
@@ -48,14 +48,19 @@ chown_pgdata:
       wal_log_hints: false
       max_wal_senders: 5 # NOTE: Since postgresql-10, the default is 10
       log_autovacuum_min_duration: '500ms'
+      log_statement: 'all'
```

Comment:
We should default to something sane here, most likely `none` (as this is the [default](https://www.postgresql.org/docs/11/runtime-config-logging.html)

Even better would be to not set a default here at all (which would automatically default to postgresql's default) and ignore the setting if unset (see below)

The same is true for the other statements you've added (especially defaulting to `pg_log` is most likely a bad idea, as logs fill up the postgresql data directory (which can result to read-only databases in case there's a lot of traffic).